app/views/devise/passwords/new.html.erb

```diff
@@ -1,16 +1,39 @@
 <%= render "devise/shared/error_messages", resource: resource %>
 
-<div class="mt-8 mb-10 crayons-card p-7 pt-3 align-left mx-auto" style="max-width:490px;">
-  <h2 class="pb-4">Forgot your password?</h2>
-  <%= form_for(resource, as: resource_name, url: password_path(resource_name), html: { method: :post }) do |f| %>
+<div class="js-new-password flex flex-col items-center s:mx-auto mx-4" style="max-width:488px;">
+  <% if Settings::General.resized_logo.present? %>
+    <a class="mt-6" href="/">
```

Comment:
Non Blocking: when i tested on mobile, the logo on this page was about 1-2 pixels lower than on the create account and login pages. not a jarring jump more of a bump. i don’t think it would be worth more time/effort than a few minutes of fiddling around with top padding

Reply:
Thanks. Fixed this for desktop as well as mobile devices across reset-password, login and signup pages.
Really appreciate pushing towards perfect design + code.